src/render/dom/utils/filter-props.ts

```diff
@@ -3,6 +3,16 @@ import { isValidMotionProp } from "../../../motion/utils/valid-prop"
 
 let shouldForward = (key: string) => !isValidMotionProp(key)
 
+export type IsValidProp = (key: string) => boolean
+
+export function loadExternalIsValidProp(emotionIsPropValid?: IsValidProp) {
```

Comment:
Why do we still call it `emotionIsPropValid` if it could be a user-provided function?